### DISCUSSION TITLE
Internal and External Perimeter Seam Alignment

### DISCUSSION THREAD

**Discussion by djngnear** [supermerill/SuperSlicer#1933] [Q&A] [answered]
How do I make sure the internal perimeter starts where the external perimeter starts when using the seam painting tool?

![SS Seam Issue 1](https://user-images.githubusercontent.com/87901234/142735858-4f6c697d-3839-44f1-bbd9-20a54dab6a04.JPG)
![SS Seam Issue 2](https://user-images.githubusercontent.com/87901234/142735859-ef23a1f7-aa18-4f51-a401-397300a5d7c0.JPG)

**Comment by supermerill** [comment, marked as answer]
You can't do that with seam painting.
But you can add a seam position sphere: right-clic, -> add seam position.
You can scale down the sphere, the seam of all perimeter will try to be the nearest possible to the x,y of the center of the sphere.
The sphere used at a layer is the sphere with the nearest z-position of its center.

**Reply by djngnear** [reply]
That worked! Thanks!
